akka-actor/src/main/scala/akka/util/Unused.scala

```diff
@@ -12,6 +14,15 @@ import akka.annotation.InternalApi
  * or other reason. Useful in combination with
  * `-Ywarn-unused:explicits,implicits` compiler options.
  *
+ * Extends 'deprecated' to make sure using a parameter marked @unused
+ * produces a warning, and not using a parameter marked @unused does not
+ * produce an 'unused parameter' warning.
+ *
+ * Extending 'deprecated' is deprecated in Scala 2.13 and scheduled to be
```

Comment:
What's the alternative? Dropping deprecated and losing the check it isn't on a used param?

Reply:
Yes (or using `@deprecated` everywhere we're currently using `@unused` but that seems worse)